modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/consensus/ConsensusManager.scala

```diff
@@ -197,6 +198,14 @@ object ConsensusManager {
       private def afterTimeTrigger: F[Unit] =
         scheduleFacility >> consensusStorage.containsTriggerEvent
           .ifM(internalFacilitateWith(EventTrigger.some), Applicative[F].unit)
+
+      private[consensus] def registeredPeers: F[List[PeerId]] =
+        (consensusStorage.getLastKey, consensusStorage.getOwnRegistration).flatMapN {
+          case (Some(lastKey), Some(_)) =>
+            consensusStorage.getRegisteredPeers(lastKey.nextN(2L)).map(selfId :: _)
```

Comment:
Why are we asking for registrations for a key +2 from the last one?

Reply:
When node is in `Observing` status it is registered to consensus with last key + 2. But as we spoke, it is safe to send blocks to it, so I want to return it as well.

Reply:
I don't see how asking for +2 changes anything. Maybe it's better to query for facilitators of the last finished consensus? If we're still in the Observing state we won't have any finished consensus yet.

Reply:
@kriscieslak you need to think about this endpoint not by its use-case but by the meaning what it provides (what API client can expect).

`/consensus/info` -> as an API client I'd expect a consensus info (registered peers may be one of its properties, but definitely not the consensus info
`/consensus/peers`  -> as an API client, I'd expect to get a list of peers taking part in a consensus
or as @tbekas suggested `/consensus/{ordinal}/registrations` -> as an API client, I'd expect to get a list of registrations

Having that in mind, there should not be peers in "Observing" as they are not part of consensus yet. correct me if I'm wrong @tbekas

Reply:
Ok, so maybe then
```
/consensus/{ordinal}/peers
```
and
```
/consensus/latest/peers